user/admin.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from .models import Account
```

Comment:
Django自体以外からimportする場合は1行空けておきましょう
```suggestion
from django.contrib import admin
from django.contrib.auth.admin import UserAdmin

from .models import Account
```